packages/contracts-bedrock/scripts/libraries/Config.sol

```diff
@@ -34,10 +34,11 @@ enum Fork {
     ECOTONE,
     FJORD,
     GRANITE,
-    HOLOCENE
+    HOLOCENE,
+    ISTHMUS
```

Comment:
Adding a note here since I found that files related to devnet such as `state-dump` etc. are generated per hard fork, based on this list. When adding a new hard fork, this list must be updated for the input files for the devnet to be generated correctly (you will see errors related to missing state dump files during the build otherwise):

See:

[stateDumpPath](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/scripts/libraries/Config.sol#L106-L112)
[dumpState](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/scripts/L2Genesis.s.sol#L639)